metals/src/main/scala/scala/meta/internal/metals/ServerCommands.scala

```diff
@@ -476,6 +476,20 @@ object ServerCommands {
     """|This command should be sent in with the LSP [`TextDocumentPositionParams`](https://microsoft.github.io/language-server-protocol/specifications/specification-current/#textDocumentPositionParams)
        |""".stripMargin,
   )
+  final case class ExtractMethodParams(
+      position: TextDocumentPositionParams,
+      applRange: Integer,
+      lv: Integer,
+  )
```

Comment:
```suggestion
  final case class ExtractMethodParams(
      textDocument: TextDocumentIdentifier,
      range: Range, // from LSP
      extractPosition: Position,
  )
```
maybe this would make it easier to handle? And also `applRange` is a bit unclear, it's better to be explicit and send in the entire range